Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+    skipDefaultCheckout()  // see 'Checkout SCM' below, once perms are fixed this is no longer needed
+  } 
+
+  stages {
+    stage('Checkout SCM') {
+      steps {
+        sh 'sudo chown -R jenkins:jenkins .'  // bad docker mount creates unreadable files TODO fix this
```

Comment:
I took it out and have run the job a couple of times without issue. I'll leave it out for now -- we can put it back if the build breaks.